### ISSUE TITLE
Accessibility criteria not met. Labels are not asscociated with checkbox

### ISSUE THREAD

**Issue by arishu39** [CuppaLabs/angular2-multiselect-dropdown#274]
Whenever we run Axe or Wave tool to check accessibility, component is giving lot of errors as Labels are not associated with checkbox in the dropdown options. Kindly look into this matter

**Comment by ganeshjangam** [comment]
angular-2-dropdown-multiselect is not passing any accessibility tests. You can use tools like Axe or Wave to check this.
Wave tool-  The issue is the labels are not associated with the checkboxes.
Axe tool - Also the upward and downward arrows are using same id. The SVG element is using the same id (Capa_1) if we have multiple dropdowns on the same page.

Also tabbing doesn't works on this dropdown.

Please look into these issues

**Comment by maheshhaval** [comment]
I am also facing the same issue with angular-2-dropdown-multiselect for accessiblity. Could you please look into this issue.
